CppCoreGuidelines.md

```diff
@@ -13167,10 +13167,10 @@ would not in itself save you.
 
 The opposite condition is most easily expressed using a negation:
 
-    // These all mean "if `p` is `nullptr`"
```

Comment:
The `\`` around the values isn't necessarily a typo; it's there to emphasis/denote the code value and demark in markdown type processing.

Reply:
Thank you for the review!
I removed ' since this part is a comment included in a code area where markdown's emphasis cannot be applied